dart/dynamics/detail/MultiDofJoint.h

```diff
@@ -37,6 +37,19 @@
 #ifndef DART_DYNAMICS_DETAIL_MULTIDOFJOINT_H_
 #define DART_DYNAMICS_DETAIL_MULTIDOFJOINT_H_
 
+#define MULTIDOFJOINT_REPORT_DIM_MISMATCH( func, arg )              \
+  dterr << "[MultiDofJoint::" #func "] mismatch beteween size of "  \
```

Comment:
Okay, then let's capitalize the the first letter. It would be good when the message is not single sentence.